lib/node_modules/@stdlib/repl/lib/completer_preview.js

```diff
@@ -246,7 +246,7 @@ setNonEnumerableReadOnly( PreviewCompleter.prototype, 'onKeypress', function onK
 * @type {Function}
 * @param {string} data - input data
 * @param {(Object|void)} key - key object
-* @returns {void}
+* @returns {boolean} boolean indicating whether the preview was auto-completed
```

Comment:
The method still returns `void` in a bunch of cases, so this JSDoc annotation should be updated. Looks like `@returns {(boolean|void)}` would reflect current implementation.